analysers/analyser_merge_defibrillators_FR_toulouse.py

```diff
@@ -31,11 +31,12 @@ def __init__(self, config, logger = None):
             u"https://www.data.gouv.fr/fr/datasets/localisation-des-defibrillateurs-toulouse/",
             u"Localisation des défibrillateurs - Toulouse",
             GeoJSON(Source(attribution = u"data.gouv.fr:Toulouse métropole",
-                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/0feb0a63-cb64-40a9-927f-628288f257e3")),
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/0feb0a63-cb64-40a9-927f-628288f257e3"),
+                ),
```

Comment:
Please keep as is.
https://github.com/frodrigo/osmose-backend/commit/6a064ad38654130f093bc7a465ac21f3b898d5fb
https://github.com/frodrigo/osmose-backend/commit/70233e3f5bf6e208b684c954447e24da540163a4